packages/insomnia-app/app/common/import.ts

```diff
@@ -77,8 +77,7 @@ export async function importUri(uri: string, importConfig: ImportRawConfig) {
   if (error) {
     showError({
       title: 'Failed to import',
-      // @ts-expect-error -- TSCONVERSION appears to be a genuine error
-      error: error.message,
+      error: error,
```

Comment:
```suggestion
      error,
```
I suppose maybe now is the time to introduce https://eslint.org/docs/rules/object-shorthand

Reply:
(well, not this PR, I just mean, like, some time these days)